knucleotide/knucleotide-fast.jl

```diff
@@ -10,9 +10,6 @@
 using Distributed
```

Comment:
Should the changes to this file be cherry-picked out of this branch as an improvement to the existing Julia program?